CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+Thank you for your interest in contributing to Glosario!
+We welcome contributions of all kinds.
+If you're here to submit a definition,
+either for a new term
+or in another language for an existing term,
+please take a moment to read the guidance below.
+
+## Contribution Workflow
+
+When you're making your contribution(s) to Glosario, please keep the following
+guidance in mind:
+
+1. **Check the [Issues page of the Glosario repository][issues]** to see what terms and definitions other contributors are already working on. This will help you avoid accidentally spending time writing the same term(s)/definition(s) as someone else.
+2. When you know what term or definition you'd like to add, tell others about it by writing **a [new issue] on the main Glosario repository.**
+3. **Add one new term or definition per Pull Request.** We'd love to receive multiple contributions from you but please keep each separate new term or definition in a distinct branch (see below) and Pull Request. This makes it much easier for the Glosario maintainers/editors to review your contributions. When you open the Pull Request you can refer to the Issue you opened before (see step 2),  e.g. if your Issue was number 49 you should write "Closes \#49." GitHub will then automatically close the Issue if and when your Pull Request is merged.
```

Comment:
Again, I suspect this is too heavyweight: my experience is that I usually need to add 2-4 terms at a time because (for example) "anti-join" and "outer join" depend on "join". Creating three PRs would be possible in this case, but one-per-term would _not_ work in the case where definitions refer to each other: if A references B and B references A and they're both new, a single-term PR will fail a glossary check.

Reply:
> Again, I suspect this is too heavyweight: my experience is that I usually need to add 2-4 terms at a time because (for example) "anti-join" and "outer join" depend on "join". Creating three PRs would be possible in this case, but one-per-term would _not_ work in the case where definitions refer to each other: if A references B and B references A and they're both new, a single-term PR will fail a glossary check.

I think that's a fair point. Maybe rewording this to say "Add one or a small cluster of related terms per pull request" with an upper limit of, say, ten terms? Since we are using this as a pedagogical tool, I want to reinforce the concept of making smaller pull requests to reduce the burden on both contributors and editors.